tools/run-tests-by-name-regex.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
I think this file got accidentally re-added. I deleted it a few commits back and just put the logic inline in `runtime/Makefile` for simplicity.